Dockerfile

```diff
@@ -0,0 +1,7 @@
+# API Dockerfile
+
+FROM python:3.9-alpine
+COPY requirements.txt .
+RUN pip3 install -r requirements.txt && rm requirements.txt
```

Comment:
@Damantino why do you think we should remove `requirements.txt ` from our container?

Reply:
I think that since it's not used anymore why have it there? That was my logic. We have it on the repo anyways.

Reply:
@Damantino I see, thank you. I just haven't seen such implementation even and I'm not sure if I see the profit here. 

Let's imagine, for some reason, you are still developing/debugging the code inside the container and for some time you might be in need of that `requirements.txt` just in case to reinstall/modify the existing libs or its version.

And, consider having a `requirements.txt` inside costs you nothing I would rather keep it for a good in the container.
cc @DominicFrei

Reply:
That is indeed a good point. The `requirements.txt` is part of a python project, independent of it being inside or outside a container. If there isn't a problem in having it there I would strongly tend with @iamtodor's explanation and opinion of keeping it where it is. @Damantino

Reply:
@Damantino could you please remove `&& rm requirements.txt` from `Dockerfile` so then we can count on merging the PR to the project, right @DominicFrei?

Reply:
:+1:

Reply:
Sure, I see your point and I completely agree. I will remove the remove of requirements 👍